fetchmail_attach_from_folder/views/fetchmail_server.xml

```diff
@@ -61,50 +61,67 @@
-                                        placeholder="[('state', '=', 'open')]"
-                                    />
-                                    <field
-                                        name="model_order"
-                                        placeholder="name asc"
-                                        attrs="{'readonly':
-                                            [('match_first','==',False)],
-                                            'required':
-                                            [('match_first','==',True)]}"
-                                    />
-                                    <field name="flag_nonmatching" />
+
+                                <!-- Matching Options -->
+                                <group>
+                                    <separator string="Matching Options" />
+                                    <group colspan="2">
+                                      <field
+                                            name="model_id"
+                                            widget="many2one"
+                                            options="{'no_create': True}"
+                                            nolabel="0"
+                                        />
+                                      <field name="match_algorithm" />
+                                    </group>
+                                    <group
+                                        attrs="{'invisible': [('match_algorithm','=','odoo_standard')]}"
+                                        colspan="2"
+                                    >
+                                        <field
+                                            name="model_field"
+                                            placeholder="email"
+                                            attrs="{'required': [('match_algorithm','in',['email_exact','email_domain'])]}"
+                                        />
+                                        <field
+                                            name="mail_field"
+                                            placeholder="to,from"
+                                            attrs="{'required': [('match_algorithm','in',['email_exact','email_domain'])]}"
+                                        />
+                                        <field
+                                            name="domain"
+                                            placeholder="[('state', '=', 'open')]"
+                                        />
+                                        <field name="match_first" />
+                                        <field
+                                            name="model_order"
+                                            placeholder="name asc"
+                                            attrs="{'readonly': [('match_first','==',False)],                                                     'required': [('match_first','==',True)]}"
```

Comment:
Not sure what happened here, but the string starting with `'required'`could go on the next line, under attrs (or under readonly).